### PR TITLE
Bookstore: create and remove books

### PR DESCRIPTION
**Pull request by VSeray**
- Configure the Redux Store and import the necessary methods from Redux:
- Write the book's actions and reducer using the ducks' pattern.
- Define Action Types, Action Creators, reducer, and initiate default state.
- Set the data inputs in the local React state for adding new books.
- Implement functionality for dispatching a corresponding action for removing books.
- Use the redux-logger to check for state changes.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
```
Comment (mikethreels) [inline src/redux/books/books.js:21]: - [ ] when adding a book it doesn't actually update the Redux store with the new book so after refreshing the the page the placeholder books are shown again and the added books are not visible anymore.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Vana, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.payload);
```
Comment (FrederickMih) [inline src/redux/books/books.js:24]: - [ ] As pointed out by the previous reviewer, please let's make sure that adding and removing a book actually update the Redux Store. Books added disappear when the page is refreshed.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @VSeray , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response. 😄

**Review by mke2111 — COMMENTED** [review]
Hello @VSeray  👋,

_This is an additional review as a result of the second opinion request._

For this project milestone, the book should be added and removed successfully. Persistence on refreshing the page is not part of the project requirements. You may consider this request optional, but if you please, you can still go ahead and add this extra functionality.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Review by Emmyn5600 — APPROVED** [review]
Hi @VSeray 

STATUS: APPROVED :partying_face: 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

### Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**